storage/src/rocksdb_client.cpp

```diff
@@ -578,6 +578,10 @@ Status Client::rangeDel(const Sliver &_beginKey, const Sliver &_endKey) {
   return Status::OK();
 }
 
+std::string Client::getPathForCheckpoint(std::uint64_t checkpointId) const {
+  return (fs::path{dbCheckpointPath_} / std::to_string(checkpointId)).string();
```

Comment:
db_checkpoint_path

Reply:
Same as above - will keep for consistency in that file for now.